<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelRankingDialog</class>
 <widget class="QDialog" name="LevelRankingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>闯关排行榜</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f0;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4CAF50;</string>
     </property>
     <property name="text">
      <string>闯关排行榜</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutControls">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    padding: 8px;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #45a049;
}</string>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/refresh.png</normaloff>:/img/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    border-radius: 4px;
    gridline-color: #f0f0f0;
}

QTableWidget::item {
    padding: 5px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #e8f5e8;
}

QHeaderView::section {
    background-color: #4CAF50;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}

QTableWidget::item {
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:nth-child(odd) {
    background-color: #f9f9f9;
}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>排名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>昵称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最高关卡</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>更新时间</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #666666;</string>
     </property>
     <property name="text">
      <string>点击刷新获取排行榜数据</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBack">
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 6px;
    font-size: 16px;
}
QPushButton:hover {
    background-color: #d32f2f;
}
QPushButton:pressed {
    background-color: #b71c1c;
}</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>